packages/shared/src/components/post/PostContentContainer.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement, ReactNode } from 'react';
```

Comment:
There seems to be an issue with the shared post modal (no image and incorrect link)
<img width="651" alt="SCR-20230130-ew0" src="https://user-images.githubusercontent.com/1488164/215416801-c9476620-8131-417d-91f8-d0dabc7b2c81.png">

Reply:
The link should be to the article, not the post page imo

Reply:
I think this is a question suited for the product team.